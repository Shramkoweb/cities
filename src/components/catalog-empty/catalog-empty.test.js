import React from "react";
import {configure, shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import CatalogEmpty from "./catalog-empty";

configure({adapter: new Adapter()});

it(`Empty main page renders correctly`, () => {
  const EmptyCatalog = shallow(
      <CatalogEmpty currentCity={`Kiev`}/>
  );

  expect(EmptyCatalog).toMatchSnapshot();
});
