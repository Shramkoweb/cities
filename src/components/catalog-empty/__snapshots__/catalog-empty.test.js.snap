// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty main page renders correctly 1`] = `
<main
  className="page__main page__main--index page__main--index-empty"
>
  <h1
    className="visually-hidden"
  >
    Cities
  </h1>
  <WithActiveElement
    currentCity="Kiev"
  />
  <div
    className="cities"
  >
    <div
      className="cities__places-container cities__places-container--empty container"
    >
      <section
        className="cities__no-places"
      >
        <div
          className="cities__status-wrapper tabs__content"
        >
          <b
            className="cities__status"
          >
            No places to stay available
          </b>
          <p
            className="cities__status-description"
          >
            We could not find any property available at the moment in 
            Kiev
          </p>
        </div>
      </section>
      <div
        className="cities__right-section"
      />
    </div>
  </div>
</main>
`;
